docs/compass/03-packages-credential-requests.md

```diff
@@ -0,0 +1,137 @@
+# Credential requests for Packages
+
+## Overview
+
+On Runtime, Application is represented as Service Class, and every Package within Application is represented as Service Plan. This document describes credential requests for APIs during the Service Plan provisioning. It also mentions passing optional input parameters from Kyma Runtime to Application or Integration System during the provisioning process.
+
+## Assumptions
+
+- Multiple Service Instances can be created from a given Package within Application.
+- During Service Class provisioning, the provided input has to be sent back to Integration System or Application via Compass. The reason is that there is no trusted connection between Integration System and Runtime.
```

Comment:
- input is optional?
- via Compass - I suggest being more specific: via Director

this sentence is too concise IMO.

Reply:
What about now?